Loop/Managers/RemoteDataServicesManager.swift

```diff
@@ -24,6 +24,8 @@ enum RemoteDataType: String {
 final class RemoteDataServicesManager {
 
     public typealias RawState = [String: Any]
```

Comment:
There was not a reference to cgmManager here so I added a delegate to capture that.